aries-site/src/data/wcag/datasearch.json

```diff
@@ -0,0 +1,512 @@
+[
+  {
+    "rule": "1.1.1",
+    "label": "Non-text content",
+    "status": "conditional",
+    "notes": "If an icon is provided, an aria-label can be added to the icon."
```

Comment:
Wondering if this should be passing since in DataSearch we provide an icon by default that comes with an aria-label

Reply:
that is a good point I can update